lambda/es-proxy-layer/lib/hits_topic_tiebreaker.js

```diff
@@ -25,25 +25,21 @@ function is_topic_match(topic1, topic2) {
 }
 
 function sort_hits_by_topic_match(topic, hits) {
-    let i=0;
     let sorted_hits=[];
-    while (i < hits.length) {
+    for (let i=0; i < hits.length; i++) {
         if (is_topic_match(topic, hits[i]._source.t)) {
             sorted_hits.unshift(hits[i]);
         } else {
             sorted_hits.push(hits[i]);
         }
-        i++;
-    } 
-    return sorted_hits;  
+    }
+    return sorted_hits;
 }
 
 function hits_topic_tiebreaker(topic, hits) {
     console.log(`Apply topic "${topic}" to rank order top hits with matching score`);
     let equal_hits = []
-    let i=0;
-    let top_score=0;
```

Comment:
@rstrahan I'm removing this top_score variable as it isn't used, but do have a look please; was there some logic you were thinking of implementing?